modules/documents/client/views/partials/modal-document-confirm-publish.html

```diff
@@ -3,10 +3,12 @@
 		<button class="btn btn-default close" type="button" ng-click="confirmDlg.cancel()">
 			<span aria-hidden="true">&times;</span>
 		</button>
-		<h3 class="modal-title">Confirm Publish File(s)</h3>
+		<h3 ng-if="confirmDlg.publishableFiles.length === 0">No files can be published</h3>
```

Comment:
That's correct.

Change 3 lines:
<h3 ng-if="confirmDlg.publishableFiles.length === 0">No files can be published</h3>
<h3 ng-if="confirmDlg.publishableFiles.length === 1">Confirm Publish File</h3>
<h3 ng-if="confirmDlg.publishableFiles.length > 1">Confirm Publish Files</h3>

To 1 line:

<h3>{{ confirmDlg.publishableFiles | filterPublishableFiles }}</h3>

-----------------------------------------------------------
Then, write a new 

filterPublishableFiles is written as a filter injection in utils.client.filters.js (utils module):

angular.module('utils')
	.filter('filterPublishableFiles', filterPublishableFiles);

function filterPublishableFiles() {
	return function(input) {
            if (input.length === 1) {
              return "Confirm Publish File.";
            } else if (input.length > 1) {
              return "Confirm Publish Files";
            } else {
              return "No files can be published.";
            }
	};
}

You can extrapolate this design and apply it to other areas.  You could even modify it to be even more generic.